content/productivity/remote-explore/plugin.js

```diff
@@ -19,13 +19,25 @@ const { MinerManager: Miner, SwissCheesePattern, SpiralPattern, TowardsCenterPat
 
 const NEW_CHUNK = 'DiscoveredNewChunk';
 
+function getSideLength(chunkSize) {
+  switch (chunkSize) {
+    case 'large':
+      return 512;
+    case 'medium':
+      return 256;
+    default:
+      return 128;
```

Comment:
@fracek any reason to add smaller chunks? I think sometimes a local mimc-fast can benefit from 16 size chunks

Reply:
That's a good point. I think on some systems it can also benefit from chunks bigger than 512. 
I see two ways to do it:
* Use `huge = 1024, large = 512, medium = 256, small = 128, tiny = 64, smallest = 16`
* Show the chunk size directly in the dropdown: `1024, 512, 256, 128, 64, 32, 16`

Reply:
Let's show chunk sizes! 🎉

Reply:
I pushed an updated.